src/Source/C3DTilesIonSource.js

```diff
@@ -0,0 +1,51 @@
+import Source from 'Source/Source';
+import Fetcher from 'Provider/Fetcher';
+
+/**
+ * @classdesc
+ * An object defining the source connection to a 3DTiles asset of a [Cesium ion server](https://cesium.com/learn/ion/).
+ *
+ * @extends Source
+ *
+ * @property {boolean} isC3DTilesIonSource - Used to checkout whether this source is a C3DTilesIonSource. Default is
+ * true. You should not change this, as it is used internally for optimisation.
+ * @property {string} url - The URL of the tileset json.
+ * @property {string} baseUrl - The base URL to access tiles.
+ * @property {string} accessToken - The Cesium ion access token used to retrieve the resource.
+ * @property {string} assetId - The id of the asset on Cesium ion.
+ */
+class C3DTilesIonSource extends Source {
+    constructor(source) {
```

Comment:
Reading the properties lists does not tell user that only `accessToken` and `assetId` are to be passed as parameters of the `constructor`. Maybe we could add doc on which parameters the `constructor` takes ?

Reply:
yep you are right, I added more doc.

Reply:
If I understood correctly, `url` seems not to be a mandatory parameter for this constructor, since it is overwritten on line 36 before being passed to `Source` constructor.

Reply:
my bad I went to fast on updating the doc. 

Access to a tileset on Cesium ion is done in two passes:
* one query to the server with the asset id and the access token that returns among other formation the actual url of the tileset
* a second query to actually get the tileset

Therefore the URL of the first query is added as source url at line 36 (since it is mandatory for `Source` constructor) and then replaced by the URL of the actual resource at line 50.